src/components/Expertise.tsx

```diff
@@ -0,0 +1,52 @@
+import { motion } from "framer-motion";
+import Card from "./UI/Card";
+
+export default function Expertise() {
+  return (
+    <>
```

Comment:
It seems that this line is not necessary as everything is already in a section tag